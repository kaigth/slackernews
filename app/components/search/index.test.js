import React from 'react';
import Search from './index';
import renderer from 'react-test-renderer';

it('renders the Search correctly', () => {
  const tree = renderer
    .create(<Search />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});