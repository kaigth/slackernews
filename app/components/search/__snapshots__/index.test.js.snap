// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Search correctly 1`] = `
<div
  className="search"
>
  <input
    className="_input"
    onChange={[Function]}
    placeholder="Search"
    value={undefined}
  />
</div>
`;
